joint_trajectory_controller/doc/speed_scaling.rst

```diff
@@ -0,0 +1,105 @@
+Speed scaling
+=============
+
+The ``joint_trajectory_controller`` (JTC) supports dynamically scaling its trajectory execution speed.
+That means, when specifying a scaling factor :math:`{f}` of less than 1, execution will proceed only
```

Comment:
Is it possible to specify a value > 1 for simulation purposes ? We are using this quite a lot in our setup. 
What would the maximum rate be? I saw the limit with the old implementation at 2.5

Reply:
Yes, it can be greater than 1. I haven't investigated a limit, though.